src/OrchardCore/OrchardCore.Abstractions/BackgroundTasks/ShellContextExtensions.cs

```diff
@@ -17,7 +17,7 @@ public static HttpContext CreateHttpContext(this ShellContext shell)
         context.Features.Set(new ShellContextFeature
         {
             ShellContext = shell,
-            OriginalPathBase = String.Empty,
+            OriginalPathBase = PathString.Empty,
```

Comment:
Yes `PathString.Empty` does the same

    public static readonly PathString Empty = new(string.Empty);

But cleaner

Reply:
@jtkech I actually this is the root of the issue.

```
PathString a = PathString.Empty;
PathString? b = String.Empty;

a.HasValue != b.HasValue
```

reason for this is because one `PathString` and the other nullable `PathString?` 

So when I added `PathString.Empty` I am comparing `PathString` instead of `PathString?`